src/main/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLetters.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.sendletter.tasks;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.sendletter.entity.Letter;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+import uk.gov.hmcts.reform.sendletter.entity.LetterState;
+import uk.gov.hmcts.reform.slc.services.steps.getpdf.FileNameHelper;
+import uk.gov.hmcts.reform.slc.services.steps.getpdf.PdfDoc;
+import uk.gov.hmcts.reform.slc.services.steps.sftpupload.FtpClient;
+import uk.gov.hmcts.reform.slc.services.steps.sftpupload.exceptions.FtpStepException;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+
+public class UploadLetters {
```

Comment:
May be worth adding `Task` suffix so that it's easier to find in the futures.
It will also be more consistent with current naming convention (`LettersController` etc)